astropy/coordinates/baseframe.py

```diff
@@ -691,10 +695,17 @@ def isscalar(self):
         return self.has_data and self.data.isscalar
 
     @classmethod
-    def get_frame_attr_names(cls):
+    def get_frame_attr_defaults(cls):
+        """Return a dict with the defaults for each frame attribute"""
         return {name: getattr(cls, name)
                 for name in cls.frame_attributes}
 
+    @deprecated("5.2", alternative="get_frame_attr_defaults or frame_attributes.keys()")
```

Comment:
`alternative` should be the actual alternative of the method, not the alternative of what the deprecated method name would suggest. If you do wish to explicitly mention how to get the actual frame attribute names then use a custom warning message with the `message` argument.
```suggestion
    @deprecated("5.2", alternative="get_frame_attr_defaults")
```